fe/fe-core/src/main/java/org/apache/doris/binlog/TableBinlog.java

```diff
@@ -305,10 +338,16 @@ public void getBinlogInfo(Database db, BaseProcResult result) {
             info.add(lastBinlogCommittedTime);
             info.add(readableLastBinlogCommittedTime);
             String binlogTtlSeconds = null;
+            String binlogMaxBytes = null;
+            String binlogMaxHistoryNums = null;
             if (binlogConfig != null) {
                 binlogTtlSeconds = String.valueOf(binlogConfig.getTtlSeconds());
+                binlogMaxBytes = String.valueOf(binlogConfig.getMaxBytes());
```

Comment:
Should we set a default max bytes as 5% of xmx?

Reply:
Based on the known information, it seems that using a time-based gc is already effectively controlling memory usage. Directly limiting memory usage to a maximum of only 5% would introduce other variables and compromise stability. It is recommended to maintain consistency with previous behavior for now.